### PR TITLE
Add Sidekiq 5.0 support

### PR DESCRIPTION
**Pull request by bluurn**
issue: https://github.com/remind101/request_id/issues/22

### CODE REVIEW

File: spec/support/sidekiq.rb [hunk 1]
```diff
@@ -6,19 +6,15 @@ def self.logger
   end
 end
 
-# https://github.com/mperham/sidekiq/blob/602d5da96d0101f47d7e89602478b2246853733e/lib/sidekiq/middleware/server/logging.rb
+https://github.com/mperham/sidekiq/blob/aa9745e9b29d408ac423356af50859cb9c382135/lib/sidekiq/job_logger.rb
```
Comment (coralieco) [inline spec/support/sidekiq.rb]: This line should not be a comment ? `#`

Comment (bluurn) [reply spec/support/sidekiq.rb]: Oh, nice catch :) I accidentally removed this comment symbol before I have committed the changeset.

**Comment by bluurn** [comment]
@coralieco Feedback addressed, thanks!

**Comment by rex-remind101** [comment]
no longer maintained https://github.com/remind101/request_id/issues/19#issuecomment-786815719

**Pull request closed without merging**